code/modules/hydroponics/grown/replicapod.dm

```diff
@@ -94,14 +97,45 @@
 						var/mob/dead/observer/O = M
 						if(!O.can_reenter_corpse)
 							break
-					make_podman = 1
+					make_podman = TRUE
+					// Devil code
 					if(isliving(M))
 						var/mob/living/L = M
 						make_podman = !L.hellbound
 					ckey_holder = M.ckey
 					break
 
-	if(make_podman)	//all conditions met!
+	// No podman player, give one or two seeds.
+	if(!make_podman)
+		// Prevent accidental harvesting. Make sure the user REALLY wants to do this if there's a chance of this coming from a living creature.
+		if(mind || ckey)
+			if(alert("The pod is currently devoid of soul. There is a possibility that a soul could claim this creature, or you could harvest it for seeds.", "Harvest Seeds?", "Harvest Seeds", "Cancel") == "Cancel")
+				return result
+
+		// If this plant has already been harvested, return early.
+		// parent.update_tray() qdels this seed.
+		if(QDELETED(src))
+			to_chat(user, text = "This pod has already had its seeds harvested!", type = MESSAGE_TYPE_INFO)
+			return result
+
+		// Make sure they can still interact with the parent hydroponics tray.
+		if(!parent.can_interact(user))
+			to_chat(user, text = "You are no longer able to harvets the seeds from [parent]!", type = MESSAGE_TYPE_INFO)
+			return result
+
+		var/seed_count = 1
+		if(prob(getYield() * 20))
+			seed_count++
+		var/output_loc = parent.Adjacent(user) ? user.loc : parent.loc //needed for TK
+		while(seed_count)
+			var/obj/item/seeds/replicapod/harvestseeds = src.Copy()
+			result.Add(harvestseeds)
+			harvestseeds.forceMove(output_loc)
+			seed_count--
+		parent.update_tray()
+		return result
+
+	// All conditions met!
```

Comment:
You removed the check but fix intendation
This is all unreachable

Reply:
oop